bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -599,20 +304,20 @@ synchronized Future<?> submitAuditTask() {
                             bookiesToBeAudited);
                     if (auditTask != null && auditTask.cancel(false)) {
                         auditTask = null;
-                        numDelayedBookieAuditsCancelled.inc();
+                        auditorStats.getNumDelayedBookieAuditsCancelled().inc();
                     }
-                    startAudit(false);
+                    auditorBookieCheckTask.startAudit(false);
                     bookiesToBeAudited.clear();
                     return;
                 }
                 if (auditTask == null) {
                     // if there is no scheduled audit, schedule one
                     auditTask = executor.schedule(() -> {
-                        startAudit(false);
+                        auditorBookieCheckTask.startAudit(false);
```

Comment:
It's odd. The auditorBookieCheckTask is runnable, here we define a runnable to invoke `auditorBookieCheckTask.startAudit(false)`.

Reply:
auditorBookieCheckTask is runnable, and scheduled operation is a complete task execution. The runTask method is also a part of the task. startAudit is only the execution of one of the methods, and does not represent a complete scheduled task.

Reply:
Ok, that's fine.